### PR TITLE
Add new C# SDK

### PR DESCRIPTION
**Pull request by m-kus**
Preview: https://pr1155--matrix-org-previews.netlify.app

### CODE REVIEW

**Comment by m-kus** [comment]
@afranke the workflow still does not run, I assume it requires an approval

File: gatsby/content/projects/sdks/matrix-csharp-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: projectimage
+title: Matrix C# SDK
+categories: 
+ - sdk
+description:
```
Comment (afranke) [inline gatsby/content/projects/sdks/matrix-csharp-sdk.mdx:6]: Can you add a short description here?

**Review by afranke — APPROVED** [review]
Looks good to me, thanks.

**Comment by m-kus** [comment]
Thank you!

**Pull request merged**